apidoc/Titanium/UI/TabbedBar.yml

```diff
@@ -43,14 +41,19 @@ properties:
   - name: style
     summary: Style of the tabbed bar.
     description: |
-        Specify one of the constants defined in 
+        Specify one of the constants:
+        For iOS: 
         [Titanium.UI.iOS.SystemButtonStyle](Titanium.UI.iOS.SystemButtonStyle),
         either `PLAIN`, `BORDERED`, or `BAR`.
         
         The `BAR` style specifies a more compact style and allows the bar's background
         color or gradient to show through.
+        For Android:
+        [Titanium.UI.TabbedBar.TABBED_BAR_STYLE_*](Titanium.UI.TABBED_BAR_STYLE.*)
+        In Android [style](Ti.UI.TabbedBar.style) is only supported in the creation dictionary 
+        of the proxy.
     type: Number
-    default: Titanium.UI.iOS.SystemButtonStyle.PLAIN
+    default: Titanium.UI.iOS.SystemButtonStyle.PLAIN for iOS, Ti.UI.TabbedBar.TABBED_BAR_STYLES_DEFAULT for Android
```

Comment:
We need to update the docs to reference the `Ti.UI.Android.TABBED_BAR_STYLE_*` constants.

Reply:
I think this is the last change needed. @ypbnv, if you can do this by tomorrow, then that would be great. QE is ready to click that merge button. :)